molecule/resources/playbooks/pre-tasks.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Update apt cache
+  apt:
+    update_cache: yes
+    cache_valid_time: 600
+  when: ansible_os_family == 'Debian'
+
+- name: Ensure sshd is installed
+  package:
+    name:
+      - openssh-server
+    state: present
+
+- name: Ensure sshd service is started
```

Comment:
maybe it would be better to make it parametrized per distro/codename?
aloo 14.04 is really dead.